pkg/controller/sparkapplication/controller.go

```diff
@@ -177,7 +177,7 @@ func (c *Controller) Stop() {
 // Callback function called when a new SparkApplication object gets created.
 func (c *Controller) onAdd(obj interface{}) {
 	app := obj.(*v1beta2.SparkApplication)
-	glog.Infof("SparkApplication %s/%s was added, enqueueing it for submission", app.Namespace, app.Name)
+	glog.Infof("SparkApplication %s/%s was added, enqueuing it for submission", app.Namespace, app.Name)
```

Comment:
Both flavors are used in this corpus. [Google Ngrams](https://books.google.com/ngrams/graph?content=enqueueing%2C+enqueuing&year_start=1800&year_end=2019&corpus=26&smoothing=3&direct_url=t1%3B%2Cenqueueing%3B%2Cc0%3B.t1%3B%2Cenqueuing%3B%2Cc0), enqueuing is winning.